scripts/prepare_sector_network.py

```diff
@@ -2387,6 +2411,79 @@ def add_biomass(n, costs):
             p_nom_extendable=True,
         )
 
+    if biomass_potentials.filter(like="unsustainable").sum().sum() > 0:
+
+        # Create timeseries to force usage of unsustainable potentials
+        e_max_pu = pd.Series(1, index=n.snapshots)
+        e_max_pu.iloc[-1] = 0
+
+        n.madd(
+            "Store",
+            spatial.gas.biogas,
+            suffix=" unsustainable",
+            bus=spatial.gas.biogas,
+            carrier="unsustainable biogas",
+            e_nom=unsustainable_biogas_potentials_spatial,
+            marginal_cost=costs.at["biogas", "fuel"],
+            e_initial=unsustainable_biogas_potentials_spatial,
+            e_nom_extendable=False,
+            e_max_pu=e_max_pu.rename(
+                spatial.gas.biogas[0] + " unsustainable"
+            ).to_frame(),
+        )
+
+        n.madd(
+            "Store",
+            spatial.biomass.nodes,
+            suffix=" unsustainable",
+            bus=spatial.biomass.nodes,
+            carrier="unsustainable solid biomass",
+            e_nom=unsustainable_solid_biomass_potentials_spatial,
+            marginal_cost=costs.at["fuelwood", "fuel"],
+            e_initial=unsustainable_solid_biomass_potentials_spatial,
+            e_nom_extendable=False,
+            e_max_pu=e_max_pu.rename(
+                spatial.biomass.nodes[0] + " unsustainable"
+            ).to_frame(),
+        )
+
+        n.madd(
+            "Bus",
+            spatial.biomass.bioliquids,
+            location=spatial.biomass.locations,
+            carrier="unsustainable bioliquids",
+            unit="MWh_LHV",
+        )
+
+        n.madd(
+            "Store",
+            spatial.biomass.bioliquids,
+            suffix=" unsustainable",
+            bus=spatial.biomass.bioliquids,
+            carrier="unsustainable bioliquids",
+            e_nom=unsustainable_liquid_biofuel_potentials_spatial,
+            marginal_cost=costs.at["biodiesel crops", "fuel"],
+            e_initial=unsustainable_liquid_biofuel_potentials_spatial,
+            e_nom_extendable=False,
+            e_max_pu=e_max_pu.rename(
+                spatial.biomass.bioliquids[0] + " unsustainable"
+            ).to_frame(),
+        )
+
+        n.madd(
```

Comment:
How is this working with the myopic code? Should there be a lifetime for the links until when they are phased out?

Reply:
Based on our discussion: I think it's fine to keep as is.

Reply:
With infinite lifetime the links are not transferred to the next planning horizon.